src/components/RadioButtonGroup/styled/buttonItem.js

```diff
@@ -1,8 +1,19 @@
 /* stylelint-disable max-line-length */
 import styled from 'styled-components';
 import { BORDER_RADIUS_2 } from '../../../styles/borderRadius';
+import getTheme from '../../../styles/helpers/getTheme';
 
-const StyledButtonItem = styled.span`
+const StyledButtonItem = styled.span.attrs(props => {
+    const theme = getTheme(props);
+    const { getContrastText, brand } = theme.palette;
+    const { main: brandMainColor, dark: brandDarkColor } = brand;
+
+    return {
+        brandMainColor,
+        brandDarkColor,
+        getContrastText,
+    };
```

Comment:
If any of this variables are being used then remove all this and then not use attrs at all.